src/chainstate/stacks/boot/pox.clar

```diff
@@ -0,0 +1,754 @@
+;; The .pox contract
+;; Error codes
+(define-constant ERR-STACKING-UNREACHABLE 255)
```

Comment:
in idiomatic Clarity, are we capitalising constants?

Reply:
I personally prefer capitalizing constants, since that's what we do in the current codebase (and it's what I've always personally done), but if there's a Clarity coding convention established somewhere, then this code should be updated to follow it ;)

Reply:
@kantai any take on this one? I have a preference for lowercase but I'm fine either way however since these contracts are coming from the blockchain team, I would love to see this contracts being written in "idiomatic Clarity" :)

Reply:
I think upper case for constants is a good convention, though I slightly prefer upper-case and underscores rather than upper-case and dashes.

Reply:
Sounds good to me, thanks!

Reply:
In Clarity, symbol bindings are immutable, so it's not necessary to visually distinguish binding symbols to literals aka "constants".

Reply:
That's true

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1850/commits/195250fb8e4f50898753bc63303c069932b2d698